helm/fluent-logshipping-app/values-s3.yaml

```diff
@@ -1,12 +0,0 @@
-fluentd:
-  aws:
-    kiam: true
-    S3:
-      enabled: true
-      bucketName: "my-cluster-logs"
-      bucketRegion: "eu-central-1"
-      bucketPathPrefix: "gs-"
-      account: "00000000000"
-      role: "my-role"
-      s3_object_key_format: "%{path}%{time_slice}_%{index}.%{file_extension}"
```

Comment:
Why removing this? It seems like a good example, of how to use those values.

Reply:
I added it back :)